### PR TITLE
Initial update to WOW 11

### PR DESCRIPTION
**Pull request by Baschtl77**
This is the initial update to make rewatch run again under WOW 11. 
It ran at the cost of some functionality, as i just commented non-working features out:
- tooltips
- fading when player is out of range
- cooldown on spellbar

### CODE REVIEW

File: Rewatch.lua [hunk 1]
```diff
@@ -166,7 +166,7 @@ end
 -- display a debug message
 function Rewatch:Debug(message)
 
-	if(not rewatch.debug) then return end
+	--if(not rewatch.debug) then return end
```
Comment (coenvdwel) [inline Rewatch.lua:169]: when done debugging, you may want to put this back in 😁

**Review by coenvdwel — COMMENTED** [review]
<3 <3 <3 <3

File: Rewatch.lua (hunk 1 above)
Comment (Baschtl77) [reply Rewatch.lua:169]: Great, was really wondering how i enabled the debug messages :D

**Comment by Baschtl77** [comment]
Do i have to close the pull-request now?

**Comment by coenvdwel** [comment]
Checked to confirm all your changes are in `master` after pulling from Curse, all is well and merged in! 😎 💘

**Pull request closed without merging**